engine/src/test/java/io/camunda/zeebe/engine/processing/job/FailJobTest.java

```diff
@@ -336,7 +336,28 @@ public void shouldTruncateErrorMessage() {
     // then
     Assertions.assertThat(failedRecord).hasRecordType(RecordType.EVENT).hasIntent(FAILED);
 
-    final String expectedErrorMessage = "*".repeat(MAX_MESSAGE_SIZE).concat("...");
+    final String expectedErrorMessage = "*".repeat(DEFAULT_MAX_ERROR_MESSAGE_SIZE).concat("...");
+    assertThat(failedRecord.getValue().getErrorMessage()).isEqualTo(expectedErrorMessage);
+    assertThat(incident.getValue().getErrorMessage()).isEqualTo(expectedErrorMessage);
+  }
+
+  @Test
+  public void shouldNotTruncateErrorMessage() {
+    // given
+    final Record<JobRecordValue> job = ENGINE.createJob(jobType, PROCESS_ID);
+    final String errorMessage = "*".repeat(DEFAULT_MAX_ERROR_MESSAGE_SIZE);
+
+    // when
+    final Record<JobRecordValue> failedRecord =
+        ENGINE.job().withKey(job.getKey()).withErrorMessage(errorMessage).fail();
+
+    final Record<IncidentRecordValue> incident =
```

Comment:
🙃 Very nitpicky, don't feel pressured to change this if you don't want to, but we could use `var` to make it fit on a single line.

Reply:
Yes, make sense. Thank you